packages/containerd.rb

```diff
@@ -0,0 +1,55 @@
+# Adapted from Arch Linux containerd PKGBUILD at:
+# https://github.com/archlinux/svntogit-community/raw/packages/containerd/trunk/PKGBUILD
+
+require 'package'
+
+class Containerd < Package
+  description 'An open and reliable container runtime'
+  homepage 'https://containerd.io/'
+  version '1.6.1'
+  license 'Apache'
+  compatibility 'all'
+  source_url 'https://github.com/containerd/containerd.git'
+
+  binary_url({
+    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/containerd/1.6.1_armv7l/containerd-1.6.1-chromeos-armv7l.tar.zst',
+     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/containerd/1.6.1_armv7l/containerd-1.6.1-chromeos-armv7l.tar.zst',
+     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/containerd/1.6.1_x86_64/containerd-1.6.1-chromeos-x86_64.tar.zst'
+  })
+  binary_sha256({
+    aarch64: '3560445ba0e85f47cede9836cdcbfd34879251f60abb9ddb71cf8ad8e0cf403a',
+     armv7l: '3560445ba0e85f47cede9836cdcbfd34879251f60abb9ddb71cf8ad8e0cf403a',
+     x86_64: '6924a4ebf2d3f50a2e9d7f18e3c23a1733d9ad1441a5115e0810a070bc5a16e6'
+  })
+
+  git_hashtag "v#{version}"
```

Comment:
Move to above the binaries?

Reply:
One of the scripts I use keeps screwing this up. :/